.github/workflows/cleanup.yml

```diff
@@ -67,3 +64,66 @@ jobs:
               echo "Deleting container group: ${container}"
               az resource delete --ids ${container}
           done
+
+  cleanup-hsm:
+    name: Cleanup HSM Keys
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log into Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.MANAGED_ID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MANAGED_ID_TENANT_ID }}
+          subscription-id: ${{ vars.SUBSCRIPTION }}
+
+      - name: Delete HSM Keys
+        run: |
+          az keyvault key list --hsm-name ${{ vars.HSM_NAME }} \
+            --query "[].kid" -o tsv | \
+            xargs -I{} az keyvault key delete --id {}
+
+  cleanup_registry:
```

Comment:
Yeah so the first job is going to be superceded by these new tests, instead of uploading images to a registry which the examples repo pulls to test, testing will happen here. I was imagining a separate cleanup PR but since this one completes the replacement I'll remove the old stuff now

Reply:
Done in 34970d2f68d49a242b0c011d3531baa6e9b3987f